nucliadb_node/src/bin/writer.rs

```diff
@@ -234,8 +241,7 @@ pub fn get_node_metadata() -> NodeWriterMetadata {
 
         match shard.get_info(&GetShardRequest::default()) {
             Ok(count) => {
-                node_metadata.shard_count += 1;
-                node_metadata.paragraph_count += count.paragraphs as u64;
+                node_metadata.new_shard(shard.id, count.paragraphs as u64);
```

Comment:
can be done without a block:
```rust
Ok(count) => node_metadata.new_shard(shard.id, count.paragraphs as u64),
```

Reply:
Done 807bf855cfe52417caf208cb76ee1b30f2c7598c